source/Sashimi.Tests/ActionHandlersFixture.cs

```diff
@@ -237,8 +238,7 @@ public void ExtraInitParametersAreSet()
         [Test]
         public void AllowPluginDownloadsShouldBeDisabled()
         {
-            IgnoreIfVersionIsNotInRange("0.11.15", "0.15.5");
-
+            IgnoreIfVersionIsNotInRange("0.11.15", "0.15.0");
```

Comment:
I moved this back to 0.15.0 based on discussions in #34 as the intent here is for this test to only run if the terraform version is >0.11.5 and <0.15 as the support changed in 0.15.0. Running CLI version 0.15.0 against this would have caused the test to run fail as it would have been in range of >0.11.15 to <0.15.5 when it shouldn't run.